partners/partner-space/how-to/edit-client-information.mdx

```diff
@@ -0,0 +1,27 @@
+---
+meta:
+  title: How to edit client information
+  description: This page explains how to edit client information the Partner Space.
+content:
+  h1: How to edit client information
+  paragraph: This page explains how to edit client information from the Partner Space.
+tags: partner-space 
+dates:
+  validation: 2024-05-27
+  posted: 2024-05-27
+categories:
+  - partner-space
+---
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- Scaleway Partner status
+
+1. Click the **See more** button (<Icon name="more" />) to the right of the client’s creation date. 
```

Comment:
I think there should be a step to say how to get to this (the client's creation date)